ui/src/workflows/components/workflow-artifacts.tsx

```diff
@@ -40,21 +40,34 @@ export function WorkflowArtifacts(props: Props) {
     return (
         <div className='white-box'>
             <div className='white-box__details'>
+                <div className='row header'>
+                    <div className='columns download'>Download</div>
```

Comment:
The download symbol in each row is self-explanatory enough.
we probably don't need this download column, or download column name, what do you think?